frontend/packages/dev-console/src/components/import/serverless/ServerlessSection.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { connectToFlags } from '@console/internal/reducers/features';
+import { FLAG_KNATIVE_SERVING } from '@console/knative-plugin';
+import FormSection from '../section/FormSection';
+import { CheckboxField } from '../../formik-fields';
+import './ServerlessSection.scss';
+
+type ServerlessSectionProps = {
+  flags: { [key: string]: boolean };
+};
+
+const ServerlessSection: React.FC<ServerlessSectionProps> = ({ flags }) => {
+  if (flags[FLAG_KNATIVE_SERVING]) {
+    return (
+      <FormSection title="Serverless Options" divider>
+        <div className="pf-c-button odc-serverless-tech-preview">Tech Preview</div>
```

Comment:
I think you should be using PF4 label component here.

Reply:
I updated it based on direction from @serenamarie125 to use 
`<span className="label label-warning">Tech Preview</span>`